backend/routes/index.js

```diff
@@ -1,9 +1,12 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const FeatureFlagModel = require('../models/FeatureFlag.js');
+const router = express.Router();
 
-/* GET home page. */
+/* GET FeatureFlags */
 router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+  FeatureFlagModel.find({}, function(err, flags) {
+    res.json(flags);
+ });
 });
```

Comment:
Would suggest the same here as above. Also, do we need `next`? If I'm not wrong you need it when you are making middle-wares.

Reply:
As of now, I don't think so we need `next`. Let me think analyse a bit more and update the code (only if require)